homeassistant/components/zha/websocket_api.py

```diff
@@ -668,7 +668,8 @@ async def websocket_update_topology(
 ) -> None:
     """Update the ZHA network topology."""
     zha_gateway = get_zha_gateway(hass)
-    hass.async_create_task(zha_gateway.application_controller.topology.scan())
+    await hass.async_create_task(zha_gateway.application_controller.topology.scan())
```

Comment:
Assuming the zha authors are ok with this change.... there probably no need to create another task since this is already running in a background task
https://github.com/home-assistant/core/blob/a0d5fda4b6c502d877ed8ad1c3b440835606ac5f/homeassistant/components/websocket_api/decorators.py#L47

Reply:
The frontend also also already used callWS instead of sendWS to call this method. (returning a promise that never completed) So I assume it was the original intention.